adapters/Vungle/VungleAdapter/VungleRouter.m

```diff
@@ -164,8 +297,8 @@ - (void)vungleWillCloseAdWithViewInfo:(nonnull VungleViewInfo *)info
 
 - (void)vungleDidCloseAdWithViewInfo:(VungleViewInfo *)info placementID:(NSString *)placementID {
   id<VungleDelegate> delegate = [self getDelegateForPlacement:placementID];
-  [self removeDelegateForPlacementID:placementID];
   if (delegate) {
+    [self removeDelegate:delegate];
```

Comment:
Wouldn't it be safer to call the delegate callbacks (e.g. `didCloseAd:`) before removing the delegate?

Reply:
@joshuajplagonera Yes, I will swap the order Thanks for finding this. 🙂

Reply:
@joshuajplagonera Thanks, done. 😊